docs/how-to/deploy_sdcore_production.md

```diff
@@ -0,0 +1,328 @@
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriovdp-config
+data:
+  config.json: |
+    {
+      "resourceList": [
+        {
+          "resourceName": "intel_sriov_vfio_access",
+          "selectors": {
+            "pciAddresses": ["0000:04:00.0"]
+          }
+        },
+        {
+          "resourceName": "intel_sriov_vfio_core",
+          "selectors": {
+            "pciAddresses": ["0000:04:00.1"]
+          }
+        }
+      ]
+    }
+```
+
+Apply the manifest:
+
+```console
+sudo k8s kubectl apply -f sriovdp-config.yaml
+```
+
+Install the SR-IOV device plugin:
+
+```console
+sudo k8s kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/sriov-network-device-plugin/master/deployments/sriovdp-daemonset.yaml
+```
+
+Install the vfioveth CNI:
+
+```console
+sudo wget -O /opt/cni/bin/vfioveth https://raw.githubusercontent.com/opencord/omec-cni/master/vfioveth
+sudo chmod +x /opt/cni/bin/vfioveth
+```
+
+Create the `ipaddresspools.yaml` manifest for the static IP address pools for MetalLB,
+using the 4 not configured static IP address from the management network, to update the
+following content:
+
+```yaml
+---
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: lb-address-cos
+  namespace: metallb-system
+spec:
+  addresses:
+  - 10.201.0.3/32
+  avoidBuggyIPs: false
+  serviceAllocation:
+    namespaces:
+      - cos-lite
+
+---
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: lb-address-nms
+  namespace: metallb-system
+spec:
+  addresses:
+  - 10.201.0.4/32
+  avoidBuggyIPs: false
+  serviceAllocation:
+    namespaces:
+      - control-plane
+    serviceSelectors:
+      - matchExpressions:
+        - {key: "app.juju.is/created-by", operator: In, values: [traefik]}
+
+---
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: lb-address-amf
+  namespace: metallb-system
+spec:
+  addresses:
+  - 10.201.0.5/32
+  avoidBuggyIPs: false
+  serviceAllocation:
+    namespaces:
+      - control-plane
+    serviceSelectors:
+      - matchExpressions:
+        - {key: "app.juju.is/created-by", operator: In, values: [amf]}
+
+---
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: lb-address-upf
+  namespace: metallb-system
+spec:
+  addresses:
+  - 10.201.0.6/32
+  avoidBuggyIPs: false
+  serviceAllocation:
+    namespaces:
+      - user-plane
+```
+
+Apply the manifest:
+
+```console
+sudo k8s kubectl apply -f ipaddresspools.yaml
+```
+
+Extract the Kubernetes configuration to a file:
+
+```console
+sudo k8s config > sdcore_k8s_config
+```
+
+Transfer the resulting file to the machine used for installation.
+
+Reboot:
+
+```console
+sudo reboot
+```
+
+## Bootstrap a Juju controller
+
+The remaining steps need to be run from the installation machine.
+
+Add the Kubernetes cluster to the Juju client and bootstrap the controller:
+
+```console
+export KUBECONFIG=/path/to/sdcore_k8s_config
```

Comment:
Since we installed k8s earlier, we should know the exact path, and provide it here instead of a placeholder

Reply:
This is technically running on another machine, we tell the user to copy the file to the other machine, but don't give a specific path.